ql/src/java/org/apache/hadoop/hive/ql/optimizer/calcite/translator/ExprNodeConverter.java

```diff
@@ -203,6 +207,32 @@ public ExprNodeDesc visitCall(RexCall call) {
       for (RexNode operand : call.operands) {
         args.add(operand.accept(this));
       }
+    } else if (call.getKind() == SqlKind.SEARCH) {
```

Comment:
Most of the comments that I did in ASTConverter about the SEARCH operator apply also here. The code is also very similar so we should check if we can somehow refactor this logic.